django-stubs/contrib/postgres/search.pyi

```diff
@@ -104,9 +104,27 @@ class SearchHeadline(Func):
 class TrigramBase(Func):
     def __init__(self, expression: _Expression, string: str, **extra: Any) -> None: ...
 
+class TrigramWordBase(Func):
+    def __init__(self, expression: _Expression, string: str, **extra: Any) -> None: ...
+
 class TrigramSimilarity(TrigramBase):
     function: str
 
 class TrigramDistance(TrigramBase):
     function: str
     arg_joiner: str
+
+class TrigramWordDistance(TrigramWordBase):
+    function: str
+    arg_joiner: str
+
+class TrigramStrictWordDistance(TrigramWordBase):
+    function: str
+    arg_joiner: str
+
+class TrigramWordSimilarity(TrigramWordBase):
+    function: str
+
+class TrigramStrictWordSimilarity(TrigramWordBase):
+    function: str
```

Comment:
The `Func` base class already defines `function: str`, no need to repeat attributes already defined higher in class hierarchy.

Reply:
Sure, I can do the same thing for `arg_joiner` too.